cmd/kube-controller-manager/app/plugins.go

```diff
@@ -129,6 +132,8 @@ func ProbeControllerVolumePlugins(cloud cloudprovider.Interface, config componen
 			allPlugins = append(allPlugins, azure_dd.ProbeVolumePlugins()...)
 		case photon.ProviderName == cloud.ProviderName():
 			allPlugins = append(allPlugins, photon_pd.ProbeVolumePlugins()...)
+		case digitalocean.ProviderName == cloud.ProviderName():
```

Comment:
This has gotten out of hand, but we don't have a leg to stand on wrt saying no to this right now.